freqtrade/optimize/hyperopt_loss_profit_drawdown.py

```diff
@@ -23,7 +23,8 @@ def hyperopt_loss_function(results: DataFrame, trade_count: int, *args, **kwargs
         total_profit = results["profit_abs"].sum()
 
         try:
-            max_drawdown_abs, _, _, _, _ = calculate_max_drawdown(results, value_col="profit_abs")
+            _, _, _, _, _, max_drawdown_abs = calculate_max_drawdown(results, value_col="profit_abs")
```

Comment:
Hope it's better like this. Also left one more commented usage (L27) as another short option, can delete if is not going to be preferred.